js/src/views/common/share_window.js

```diff
@@ -46,8 +46,8 @@ var ShareWindowView = infoWindowView.extend({
     var buttons = document.querySelectorAll('.c-button');
       current = e.currentTarget;
 
-    $(buttons).removeClass('-active');
-    $(buttons).addClass('-secondary');
+    $(".share-toolbar .c-button").removeClass('-active');
+    $(".share-toolbar .c-button").addClass('-secondary');
```

Comment:
You are telling the browser to search twice the same element here. This is very expensive in terms of performance, please use  the `buttons` variable above to select the buttons and use it as many times you need. If you have any doubt about how to do it, just tell me.